### PR TITLE
Format online counts to locale

### PR DESCRIPTION
**Pull request by brianlovin**
**Status**
- [ ] WIP
- [x] Ready for review
- [ ] Needs testing

**Deploy after merge (delete what needn't be deployed)**
- hyperion (frontend)

### CODE REVIEW

**Review by mxstbr — CHANGES_REQUESTED** [review]
Let's make sure to do `Number(...).toLocaleString()` in case the count isn't returned correctly from the server so it doesn't crash.

File: src/components/profile/community.js [hunk 1]
```diff
@@ -187,7 +187,7 @@ class CommunityWithData extends React.Component<Props> {
                     <OnlineIndicator
                       offline={community.metaData.onlineMembers === 0}
                     />
-                    {community.metaData.onlineMembers} online
+                    {community.metaData.onlineMembers.toLocaleString()} online
```
Comment (PierreAndreis) [inline src/components/profile/community.js:190]:
```suggestion
                   {Number(community.metaData.onlineMembers).toLocaleString()} online
```

File: src/views/channel/index.js [hunk 2]
```diff
@@ -422,7 +422,7 @@ class ChannelView extends React.Component<Props, State> {
                       <OnlineIndicator
                         offline={channel.metaData.onlineMembers === 0}
                       />
-                      {channel.metaData.onlineMembers} online
+                      {channel.metaData.onlineMembers.toLocaleString()} online
```
Comment (PierreAndreis) [inline src/views/channel/index.js:425]:
```suggestion
                      {Number(channel.metaData.onlineMembers).toLocaleString()} online
```

**Comment by brianlovin** [comment]
> Let's make sure to do Number(...).toLocaleString() in case the count isn't returned correctly from the server so it doesn't crash.

How could this ever happen? GraphQL says that it must return a number, and if it can't return a number it will fail...

File: src/views/channel/index.js (hunk 2 above)
Comment (innnzzz6) [reply src/views/channel/index.js]: 425

File: src/components/profile/community.js (hunk 1 above)
Comment (innnzzz6) [reply src/components/profile/community.js]: 190

**Comment by brianlovin** [comment]
Ping @mxstbr

**Review by mxstbr — APPROVED** [review]
Okay okay you smartass... 😜

**Pull request merged**